entities-search/src/main/scala/io/renku/entities/search/model.scala

```diff
@@ -48,6 +48,7 @@ object model {
         name:             projects.Name,
         visibility:       projects.Visibility,
         date:             projects.DateCreated,
```

Comment:
We could also rename the `date` to `dateCreated` here or leave it for the next PR. Not sure what would work better. Probably it doesn't matter much as the next PR is something we'll do soon anyway. Maybe just worth taking a note.